egs/wsj/s5/steps/libs/common.py

```diff
@@ -66,6 +69,26 @@ def __call__(self, parser, namespace, values, option_string=None):
             setattr(namespace, self.dest, values)
 
 
+class KaldiIo(object):
```

Comment:
I think I prefer the old name "smart_open".  Even though it's lowercase and it's a class, since it behaves like the 'open' call I think it's clearer.
My problem with the "KaldiIo" name is that it seems to suggest that this somehow replicates Kaldi I/O mechanisms, which is far from the case; and the use of this name will tend to encourage a kind of "completism" where people will be tempted to add all kinds of Kaldi I/O mechansims to this object.  This is not the direction I want to take the python scripts.